### PR TITLE
Write format specification in rst + add noise model description

### PR DESCRIPTION
**Pull request by yannikschaelte**
* Rewrite in restructuredtext format, as this is more flexible.
* Add formulas describing precisely the noise models.

### CODE REVIEW

File: doc/documentation_data_format.rst [hunk 1]
```diff
@@ -0,0 +1,689 @@
...
+  that need to have different values for different measurements have the
+  structure: ``noiseParameter${indexOfNoiseParameter}_${observableId}``
+  to facilitate automatic recognition. The specific values or parameters are
+  assigned in the ``noiseParameters`` field of the *measurement table*
+  (see above). Any parameters named ``noiseParameter${1..n}_${observableId}``
+  *must* be overwritten in the measurement table.
+`
+- ``noiseDistribution`` [STRING: 'normal' or 'laplace', OPTIONAL]
+
+  Assumed noise distribution for the given measurement. Only normally or
```
Comment (dweindl) [inline doc/documentation_data_format.rst:354]: Currently rendered as quote

File: doc/documentation_data_format.rst [hunk 2]
```diff
@@ -0,0 +1,689 @@
...
+
+- ``initializationPriorType`` [STRING, OPTIONAL]
+
+  Prior types used for sampling of initial points for optimization. Sampled
+  points are clipped to lie inside the parameter boundaries specified by
+  ``lowerBound`` and ``upperBound``. Defaults to ``parameterScaleUniform``.
+
+  Possible prior types are:
+
+    - *uniform*: flat prior on linear parameters
```
Comment (dweindl) [inline doc/documentation_data_format.rst:500]: Nested list rendered as quote. Same for following list.

Comment (yannikschaelte) [reply doc/documentation_data_format.rst:500]: not sure, seems fine on sphinx, only github does this ...

File: doc/documentation_data_format.rst [hunk 3]
```diff
@@ -0,0 +1,689 @@
...
+  A name for the specific plot.
+
+- ``plotTypeSimulation`` [STRING, OPTIONAL]
+
+  The type of the corresponding plot, can be ``LinePlot``, ``BarPlot`` and ``ScatterPlot``. Default is ``LinePlot``.
+
+- ``plotTypeData`` [STRING, OPTIONAL]
+
+  The type how replicates should be handled, can be ``MeanAndSD``,
+  ``MeanAndSEM``, ``replicate` (for plotting all replicates separately), or
```
Comment (dweindl) [inline doc/documentation_data_format.rst:597]:
```suggestion
  ``MeanAndSEM``, ``replicate`` (for plotting all replicates separately), or
```

File: doc/documentation_data_format.rst [hunk 4]
```diff
@@ -0,0 +1,689 @@
...
+
+  Possible data-offsets for the independent variable (default is ``0``).
+
+- ``xLabel`` [STRING, OPTIONAL]
+
+  Label for the x-axis. Defaults to the entry in ``xValues``.
+
+- ``xScale`` [STRING, OPTIONAL]
+
+  Scale of the independent variable, can be ``lin``, ``log``, ``log10`` or `order``.
```
Comment (dweindl) [inline doc/documentation_data_format.rst:622]:
```suggestion
  Scale of the independent variable, can be ``lin``, ``log``, ``log10`` or ``order``.
```

File: doc/documentation_data_format.rst [hunk 5]
```diff
@@ -0,0 +1,689 @@
...
+
+and *must not* include:
+
+- Placeholder parameters (see ``observableParameters`` and ``noiseParameters``
+  above)
+- Parameters included as column names in the *condition table*
+- Parameters that are AssignmentRule targets in the SBML model
+
+it *may* include:
+- Any SBML model parameter that was not excluded above
```
Comment (dweindl) [inline doc/documentation_data_format.rst:422]:
```suggestion

- Any SBML model parameter that was not excluded above
```

File: doc/documentation_data_format.rst (hunk 2 above)
Comment (dweindl) [reply doc/documentation_data_format.rst:500]: okay, on checked there... would be good if it was working on both, as far as possible :)

Comment (yannikschaelte) [reply doc/documentation_data_format.rst:500]: definitely. just not sure what's happening, it seems like valid rst. I think github doesn't have math/latex support yet, e.g.

**Pull request merged**